queries/macros/test_teto_viagens.sql

```diff
@@ -0,0 +1,67 @@
+{% test teto_viagens(model) -%}
+    with
+        planejado as (
+            select distinct
+                data,
+                servico,
+                faixa_horaria_inicio,
+                faixa_horaria_fim,
+                partidas_total_planejada
+            from {{ ref("viagem_planejada") }}
+            -- from `rj-smtr.projeto_subsidio_sppo.viagem_planejada`
+            where
+                data between date("{{ var('date_range_start') }}") and date(
+                    "{{ var('date_range_end') }}"
+                )
+                and (distancia_total_planejada > 0 or distancia_total_planejada is null)
+                and (id_tipo_trajeto = 0 or id_tipo_trajeto is null)
+                and data >= date('{{ var("DATA_SUBSIDIO_V3A_INICIO") }}')
+        ),
+        viagens_remuneradas as (
+            select
+                r.data, r.servico, faixa_horaria_inicio, indicador_viagem_dentro_limite
+            from {{ model }} r
+            -- `rj-smtr-dev.abr_reprocessamento__dashboard_subsidio_sppo.viagens_remuneradas` r
+            left join
+                {{ ref("viagens_completa") }} c
+                -- `rj-smtr.projeto_subsidio_sppo.viagem_completa` c
+                using (data, id_viagem)
+            left join
+                planejado p
+                on p.data = c.data
+                and r.servico = p.servico
+                and c.datetime_partida
+                between faixa_horaria_inicio and faixa_horaria_fim
+            where
+                r.data between date("{{ var('date_range_start') }}") and date(
+                    "{{ var('date_range_end') }}"
```

Comment:
```suggestion
                r.data between date('{{ var("date_range_start") }}') and date(
                    '{{ var("date_range_end") }}'
```

                data between date('{{ var("date_range_start") }}') and date(
                    '{{ var("date_range_end") }}'